### PR TITLE
Add files-to-ignore-internal-links-in feature

### PR DESCRIPTION
**Pull request by dbadura**
**Description**

Changes proposed in this pull request:

- implementation of `files-to-ignore-internal-links-in`
- remove global variable `base-path`
- quick fix for skipped tests

**Related issue(s)**

#5 #1

### CODE REVIEW

File: pkg/config_file.go [hunk 1]
```diff
@@ -13,66 +20,119 @@ type FileConfig struct {
 
 func NewFileConfig(filePath string, config *Config) *FileConfig {
 	if config != nil {
```
Comment (tgorgol) [inline pkg/config_file.go]:
```suggestion
	if config == nil {
	  return nil
	}
```

File: pkg/config_file.go [hunk 2]
```diff
@@ -13,66 +20,119 @@ type FileConfig struct {
...
+	internalLinksToIgnore := config.InternalLinksToIgnore
+	if fileConfig != nil {
+		internalLinksToIgnore = unique(append(config.InternalLinksToIgnore, fileConfig.InternalLinksToIgnore...))
+	}
+
+	return internalLinksToIgnore
+}
+
+func getInternalIgnorePolicy(filepath string, config Config, fileConfig *FileConfig) bool {
+	internalIgnore := config.IgnoreExternal
```
Comment (tgorgol) [inline pkg/config_file.go]:
```suggestion
	internalIgnore := config.IgnoreInternal
```

Please cover this case in unit tests

File: pkg/config_file_test.go [hunk 3]
```diff
@@ -8,12 +8,12 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-func TestConfigFile(t *testing.T) {
-	t.Run("Config File", func(t *testing.T) {
-		SetBasePath("test-markdowns", false)
-
+func TestCombineConfigsForFile(t *testing.T) {
```
Comment (tgorgol) [inline pkg/config_file_test.go]:
```suggestion
func TestNewFileConfig(t *testing.T) {
```

**Review by tgorgol — CHANGES_REQUESTED** [review]
Good job, few comments from me

**Review by tgorgol — APPROVED** [review]
LGTM

**Review by kazydek — APPROVED** [review]

**Pull request merged**